src/uct/api/uct.h

```diff
@@ -1205,26 +1198,28 @@ struct uct_ep_params {
     unsigned                            path_index;
 
     /**
-     * Callback is called when the client side endpoint is bound to network
-     * device and provides related info as defined in
-     * @ref uct_cm_ep_resolve_args_t. This field is only valid if
-     * @ref uct_ep_params_t::sockaddr is set.
-     * @note It is never guaranteed that the callaback will be called. If, for
-     * example, the endpoint goes into error state before issuing the connection
-     * request, the callback will not be invoked.
+     * This callback is invoked when the remote server address provided in field
+     * @ref uct_ep_params_t::sockaddr is resolved to the local device to be used
+     * for connection establishment.
+     * @note In case of a connection error, this callback will not be invoked;
+     *       @ref uct_ep_params_t::sockaddr_cb_client with indicating error
```

Comment:
1. indicating error status -> the error code
2. `@ref uct_ep_params_t.sockaddr_cb_client` pls make sure all refs to fields are correct

Reply:
`make docs` checks `@ref` correctness by doxygen

Reply:
ok

Reply:
for example:
```
uct.h:1179: error: unable to resolve reference to `uct_ep_params_t::sockaddr_resolve_cb' for \ref command (warning treated as error, aborting now)
Exiting...
make: *** [docs/doxygen-doc/ucx.tag] Error 1
```
after changing to 
```
@ref uct_ep_params_t::cm_resolve_cb
```
it's passed

Reply:
great! so pls disregard my comments about using `.`